### PR TITLE
Navigation for login and logout

### PR DESCRIPTION
**Pull request by devcatch**

### CODE REVIEW

File: GiTracker/ViewModels/MainPageViewModel.cs [hunk 1]
```diff
@@ -36,8 +41,15 @@ public MainPageViewModel(IUnityContainer container)
 
         private void DoSlideMenuItemTapped(SlideMenuItem item)
         {
-            PresentedViewModelType = item.ScreenView;
-            PresentedViewModelTypeChanged?.Invoke(this, EventArgs.Empty);
+            if (item.ScreenView == null)
```
Comment (OlexaLe) [inline GiTracker/ViewModels/MainPageViewModel.cs]: please use Title for that condition.

**Comment by OlexaLe** [comment]
23h 0m logged on 25/9/2015 via #GiTracker

**Comment by OlexaLe** [comment]
2h 3m logged on 25/9/2015 via #GiTracker

**Comment by XamarinGaragePrivate** [comment]
2h 0m logged on 30/9/2015 via #GiTracker

**Pull request merged**